code/modules/antagonists/revolution/revolution.dm

```diff
@@ -443,15 +443,25 @@
 		if(charter_given || real_headrev.stat != CONSCIOUS)
 			continue
 		charter_given = TRUE
+		var/list/headrev_charter_goodies = list(
+		/obj/item/station_charter/revolution,
+		/obj/item/megaphone,
+		/obj/item/bedsheet/rev,
+		)
 		podspawn(list(
 			"target" = get_turf(real_headrev),
 			"style" = STYLE_SYNDICATE,
-			"spawn" = /obj/item/station_charter/revolution,
+			"spawn" = headrev_charter_goodies,
 		))
 		to_chat(real_headrev, span_hear("You hear something crackle in your ears for a moment before a voice speaks. \
 			\"Please stand by for a message from your benefactor. Message as follows, provocateur. \
 			<b>You have been chosen out of your fellow provocateurs to rename the station. Choose wisely.</b> Message ends.\""))
 
+	//Forces the next round to have a station trait that makes command rooms get trashed on roundstart
+	var/list/revolutionary_trait = list(/datum/station_trait/revolutionary_trashing) //Wrapped in a list for json encoding
+
+	rustg_file_write(json_encode(revolutionary_trait), FUTURE_STATION_TRAITS_FILE)
```

Comment:
Don't do this if the file already exists, since it means that an admin has explicitly set it